docs/editor_guide.md

```diff
@@ -94,8 +93,69 @@ these steps need to be taken:
    ```
 2. set the GitHub topics, if they haven't been set yet. Topics should include at least `lesson` and the appropriate life-cycle badge.
 
-### After acceptance
+### Performing editorial checks
+Using the checklist above, perform the basic editorial checks to ensure that the
+lesson is ready for review.
+Share the results of the checklist with the authors in the thread,
+highlighting any points that need to be addressed before the lesson can be reviewed.
+
+When the editorial checks have been satisfied,
+you can begin looking for reviewers.
+
+### Finding reviewers
+Each lesson should be reviewed by two people.
+Ideally, reviewers will provide a diversity of perspectives, representing e.g.
+different levels of expertise in the lesson topic and
+different local/domain communities.
+This helps to ensure that the lesson is accessible
+and guards against [expertise awareness gap](https://carpentries.github.io/instructor-training/04-expertise.html#mind-the-gap).
+
+So far, reviewers have been identified through a combination of:
+
+* responses to the Lab review volunteer form
+* the Editor's professional network
+* exploring the lists of 'watchers' on the repository of the lesson under review,
+  membership of any relevant Slack channels, etc
+
+Invitations to review have been made privately, by email/Slack message,
+followed by public invitation and confirmation on the review thread in this repository.
+(See examples of these posts in [the _Good Enough Practices_ lesson review thread](https://github.com/carpentries-lab/reviews/issues/24#issuecomment-1451796473).)
+
+Potential reviewers frequently ask what the time frame is for review.
+We do not have a strict policy,
+but hope to receive reviews within six weeks of confirmation.
+However, we would rather receive good reviews from reviewers who are a good fit for the lesson.
+So the Editor should encourage would-be reviewers to
+estimate a timeframe that would work for them
+and try to communicate that to the authors.
+
+#### Support for Editors from The Carpentries
+The [Curriculum Team](mailto:curriculum@carpentries.org) can provide a list of
+people who have volunteered to review lessons in the Lab,
+and can contact people on your behalf
+e.g. where you have identfied a potential reviewer from the community,
+whose details are stored in AMY.
+
+### Supporting the Review Process
+After assigning reviewers, the Editor should continue to support the process by:
+
+- answering questions from authors and reviewers
```

Comment:
is this in both the candidate repo and the issue in this repo or only one? 

is there guidance for what should/not be done in private communication vs on the repo/review issue?

Reply:
> is this in both the candidate repo and the issue in this repo or only one?

I have answered questions in the review issue and also where I am tagged in issues on the lesson repository, but those issues on the repository themselves have related to the lesson infrastructure (e.g. formatting and/or configuration issues) and I consider myself to have answered them more in my capacity as a member of the Curriculum Team rather than in the role of editor for the Lab.
As a guest editor volunteering their time I do not expect you to watch out for notifications outside of the thread. If you prefer to answer questions within the thread, please communicate that expectation to the authors and reviewers. 

> is there guidance for what should/not be done in private communication vs on the repo/review issue?

In [the Reviewer Guide](https://github.com/carpentries-lab/reviews/blob/main/docs/reviewer_guide.md#communication) we state that

>_The whole review process for a lesson is intended to take place in a single issue thread, with Reviewers posting their reviews and comments as replies to the original submission. Similarly, authors will then post their responses to Reviewers in the issue thread, and so on until the lesson is approved for inclusion in The Carpentries Lab, or the review submission is withdrawn by the author(s)._
> 
> _As a Reviewer, you are allowed - encouraged, even - to open issues and pull requests to report and address areas for improvement in the lesson you are reviewing. However, if you do this, please make sure to include links to these contributions in your review report when you post it to the thread on this repository._
> 
> _Similarly, if you communicate with the authors about the lesson and review outside of the issue thread, please make sure to summarise, link to, and/or quote these conversations within the thread._

We have not yet provided any similar guidance for editors, partly due to a lack of experience leading to concern that I haven't yet thought of all circumstances in which communication would be better done in private. 

My approach up to now has been to do everything publicly except for the initial approach to reviewers (and to you as a potential guest editor). I have also thanked reviewers privately (as well as publicly) and shared a link to a post-review feedback survey. 

After a review has completed, I have also communicated privately with authors to coordinate repo transfers, publication of blog posts, etc.

But I can imagine other circumstances where private communication could be necessary e.g. if reviewers and authors reach an impasse, I would like to have the option of private synchronous and asynchronous discussion with the parties concerned to see if I can get things unstuck. So far we have not had any such problems.